src/Plugin/BlockSettings/InlineEntity.php

```diff
@@ -148,7 +148,7 @@ public function buildAreaOptionsForm(array &$form, FormStateInterface $form_stat
     $display_options = $this->getDisplayOptions();
     $entity_type = $display_options['entity_reference']['target_type'];
 
-    $options = \Drupal::entityManager()->getViewModeOptions($entity_type);
+    $options = \Drupal::service("entity_display.repository")->getViewModeOptions($entity_type);

```

Comment:
Could you please enlist all these deprecation codes in confluence ? 
Something like : getViewModeOptions() should be called from \Drupal::service("entity_display.repository") so that we can search by `getViewModeOptions` in other projects

Reply:
Sure.